src/components/Common/Product/ProductCreate/ProductCreateComponent.jsx

```diff
@@ -0,0 +1,150 @@
+import { useState } from 'react'
+import { useNavigate } from 'react-router-dom'
+
+import s from './ProductCreateComponent.module.scss'
+
+import { uploadImages } from 'api/image'
+import { postProductAPI } from 'api/product'
+import { SmallButton } from 'components/Common/Button/Small/SmallButton'
+
+const ProductCreateComponent = () => {
+  const [name, setname] = useState('')
+  const [characterCount, setCharacterCount] = useState(0)
+  const [prices, setPrices] = useState('')
+  const [sendPrice, setSendPrice] = useState()
+  const [selectedImage, setSelectedImage] = useState(null)
+  const [imageUrl, setImageUrl] = useState('')
+  const [selectedImages, setSelectedImages] = useState([])
+  const [imagePreviews, setImagePreviews] = useState([])
+
+  const navigate = useNavigate()
+
+  const link = '/'
+
+  const handleSend = async () => {
+    if (imageUrl === '') {
+      window.alert('상품이미지를 등록해주세요!')
+      return
+    } else if (name === '') {
+      window.alert('상품명을 등록해주세요!')
+      return
+    } else if (sendPrice === undefined) {
+      window.alert('상품가격을 등록해주세요!')
+    } else {
+      const res = await postProductAPI({ link: link, itemName: name, price: sendPrice, itemImage: imageUrl })
+      navigate(-1)
+    }
```

Comment:
else if (sendPrice === undefined) {
      window.alert('상품가격을 등록해주세요!')
}
에서도 return으로 종료해버린다하면 이코드에서는 if else문을 사용할 필요가 없습니당!
```suggestion
    if (imageUrl === '') {
      window.alert('상품이미지를 등록해주세요!')
      return
    }
    if (name === '') {
      window.alert('상품명을 등록해주세요!')
      return
    } 
    if (sendPrice === undefined) {
      window.alert('상품가격을 등록해주세요!')
      return
    } 
     const res = await postProductAPI({ link: link, itemName: name, price: sendPrice, itemImage: imageUrl })
     navigate(-1)
  
```
그리고 alert띄우는 것보다 상품이밎, 상품명, 상품가격이 비어있을때 send버튼을 disableButton으로 하고 데이터가 있을때 클릭가능하게 하는게 더 좋은 로직같아요!